test/ThriftTest.thrift

```diff
@@ -67,7 +67,7 @@ typedef i64 UserId
 struct Bonk
 {
   1: string message,
-  2: i32 type
+  2: i32 bonkType
 }
```

Comment:
@Jens-G I'm sorry if my comment came across as rude. I'll change the code generator to prefix reserved names with an identifier.